src/test/java/org/opensearch/security/SystemIntegratorsTests.java

```diff
@@ -179,7 +179,7 @@ public void testInjectedUser() throws Exception {
         Assert.assertTrue(resc.getBody().contains("\"remote_address\":\"8.8.8.8:8\""));
         Assert.assertTrue(resc.getBody().contains("\"backend_roles\":[\"role1\",\"role2\"]"));
         // mapped by username
-        Assert.assertTrue(resc.getBody().contains("\"roles\":[\"opendistro_security_all_access\""));
+        Assert.assertTrue(resc.getBody().contains("\"opendistro_security_all_access\""));
```

Comment:
Was this change needed?

Reply:
Yes. because nagilum is now associated with another role called `who_am_i`, the request body won't always start with `\"roles\":[`

Reply:
Similar to the assertion above with backend roles, I think this should assert all expected roles